src/libraries/System.Net.Connections/src/System/Net/NetworkException.cs

```diff
@@ -38,16 +38,7 @@ public override void GetObjectData(SerializationInfo serializationInfo, Streamin
         /// <summary>Returns the specific kind of error.</summary>
         public NetworkError NetworkError { get; }
 
-        private static string GetExceptionMessage(NetworkError error) => error switch
-        {
-            NetworkError.EndPointInUse => SR.networkerror_addressinuse,
-            NetworkError.TimedOut => SR.networkerror_timedout,
-            NetworkError.HostNotFound => SR.networkerror_hostnotfound,
-            NetworkError.ConnectionRefused => SR.networkerror_connectionrefused,
-            NetworkError.ConnectionAborted => SR.networkerror_connectionaborted,
-            NetworkError.ConnectionReset => SR.networkerror_connectionreset,
-            NetworkError.OperationAborted => SR.networkerror_operationaborted,
-            _ => SR.networkerror_other
-        };
+        // TODO: Better exception messages
```

Comment:
Is this a pure git revert from the System.NET.Connections addition?

Reply:
No. NetworkException was a later addition. I've moved it to System.Net.Connections in this PR to preserve the usage of it there.

Reply:
Why not also move the resources and keep this as it was?

Reply:
I could have, but since it's not shipping I just skipped it for now.